test/clt-tests/core/test-skipping-plugin-loading.rec

```diff
@@ -0,0 +1,194 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -h0 -P9306 -e "SHOW BUDDY PLUGINS\G;" | grep "manticoresoftware/buddy-plugin-replace"
+––– output –––
+Package: manticoresoftware/buddy-plugin-replace
+––– input –––
+mysql -v -h0 -P9306 -e "SHOW TABLES; DROP TABLE IF EXISTS tbl; SHOW TABLES; CREATE TABLE tbl (id BIGINT, model TEXT, storage_capacity INTEGER, color string, release_year INTEGER, price FLOAT, discounted_price FLOAT, sold BOOL, date_added TIMESTAMP, product_codes MULTI, values MULTI64, additional_info JSON, vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='l2');"
```

Comment:
I believe it's better to split commands because we're using the Command Line Tool (CLT) for step-by-step execution. In cases where one line is incredibly long, we may experience potential problems while trying to maintain and troubleshoot issues in the future. Let's split each query into a single row.

Reply:
Fixed